tools/reviewer/job/impl/FirestoreTaskBase.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2018 The StartupOS Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.startupos.tools.reviewer.job.impl;
+
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.auth.oauth2.ServiceAccountCredentials;
+import com.google.startupos.common.firestore.FirestoreClient;
+import com.google.startupos.common.firestore.FirestoreClientFactory;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Arrays;
+
+public abstract class FirestoreTaskBase {
+  protected FirestoreClient firestoreClient;
+  protected FirestoreClientFactory firestoreClientFactory;
+
+  protected void initializeFirestoreClientIfNull() {
+    if (this.firestoreClientFactory == null) {
+      throw new IllegalArgumentException("this.firestoreClientFactory should be initialized");
+    }
+
+    if (this.firestoreClient == null) {
+      FileInputStream serviceAccount = null;
+      try {
+        serviceAccount =
+            new FileInputStream(
+                com.google.startupos.tools.reviewer.job.ReviewerJob.serviceAccountJson.get());
```

Comment:
Please use `import com.google.startupos.tools.reviewer.job.ReviewerJob;`

Reply:
@oferb resolved in 5fb8d19b2722c9c38e1eb528d1d9c75c5b40afae